### ISSUE TITLE
Changing payload_limit through poxa.conf

### ISSUE THREAD

**Issue by lorenzbausch** [edgurgel/poxa#123]
Is it possible to override `payload_limit` through `poxa.conf`? I tried, but here seems to a pattern missing.
I also tried `docker run` with `-e PAYLOAD_LIMIT=20000` but the limit still remains at 10000.

**Comment by edgurgel** [comment]
I think you are right. It's missing as part of the schema.exs: https://github.com/edgurgel/poxa/blob/master/config/poxa.schema.exs

I will update and release a new version 👍

**Comment by lorenzbausch** [comment]
Awesome, thank you very much for your quick bug fix.
Do you plan to relase a maintenance release at some point or should we just use the `latest` tag?

**Comment by edgurgel** [comment]
oops! I thought I had done this already! Just released 0.7.2! I also tested with `latest` it's  working as expected:

```
docker run -e PAYLOAD_LIMIT=152  -it --rm --name poxa edgurgel/poxa-automated:latest /app/poxa/bin/poxa console
==> Generated sys.config in /app/poxa/var
Erlang/OTP 20 [erts-9.2] [source] [64-bit] [smp:2:2] [ds:2:2:10] [async-threads:10] [hipe] [kernel-poll:false]


09:39:44.946 [info]  Starting Poxa, app_id: app_id on port 8080

09:39:44.947 [info]  SSL not configured/started
Interactive Elixir (1.5.2) - press Ctrl+C to exit (type h() ENTER for help)
iex(poxa@127.0.0.1)1> Application.get_all_env(:poxa)
[web_hook: "", app_secret: "secret",
 ssl: [keyfile: "", cacertfile: "", port: 8443, enabled: false, certfile: ""],
 included_applications: [], registry_adapter: "gproc", app_key: "app_key",
 app_id: "app_id", port: 8080, payload_limit: 152]
```
